doc/getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting started
+
+<!-- TOC -->
+
+- [Getting started](#getting-started)
+    - [SPIRE artifacts](#spire-artifacts)
+    - [Building the artifacts](#building-the-artifacts)
+    - [Installing your artifact in Ubuntu 16.04](#installing-your-artifact-in-ubuntu-1604)
+    - [Configuration](#configuration)
+    - [Running SPIRE with join token](#running-spire-with-join-token)
+
+<!-- /TOC -->
+
+## SPIRE artifacts
+
+You can either download the binaries from [here](https://github.com/spiffe/spire/releases) or build them by
```

Comment:
I suggest we simplify this by only describing how to install from a pre-built binaries

Reply:
I think building from source is in-scope for this doc, but i also think that the audience for this doc is mostly developers at the moment.

Reply:
There's a lot of inherent complexity in building from source - especially for someone who isn't a Go developer already. If we can avoid that complexity for the purposes of this guide, we should. It should also be easy enough for a motivated developer to find the build instructions if they want to do it the hard way.